Bottom: ddb6c356edf01b8020d228a70a140e8737dffdba
Top:    12b1e8990b3798c411666542ffc72c4e0a85d9bb
Author: Peter Clifton <pcjc2@cam.ac.uk>
Date:   2013-06-05 12:07:57 +0100

Add warning about bogus code in strflags.c


---

diff --git a/src/strflags.c b/src/strflags.c
index 3e57696..d6d6273 100644
--- a/src/strflags.c
+++ b/src/strflags.c
@@ -576,6 +576,7 @@ dump_flag (FlagType * f)
 {
   int l;
   printf ("F:%08x T:[", f->f);
+#warning THIS NEXT LINE IS BOGUS, THE ARRAY IS HALF THIS LENGTH
   for (l = 0; l < (MAX_LAYER + 7) / 8; l++)
     printf (" %02x", f->t[l]);
   printf ("]");
